e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubEdgeBaseDeployment.cs

```diff
@@ -0,0 +1,124 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestExtensions {
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Newtonsoft.Json;
+    using Microsoft.Azure.Devices;
+
+    /// <summary>
+    /// Default edge base deployment configuration
+    /// </summary>
+    public class IoTHubEdgeBaseDeployment : DeploymentConfiguration, IIotHubDeployment{
+
+        /// <summary>
+        /// Target condition for gateways
+        /// </summary>
+        public readonly string TargetCondition =
+            $"(tags.__type__ = 'iiotedge' AND tags.unmanaged = true)";
```

Comment:
Please move targetcondition to test constants

Reply:
done